lib/membrane_rtmp_plugin/rtmp_server.ex

```diff
@@ -15,7 +21,8 @@ defmodule Membrane.RTMP.Server do
           handler: ClientHandlerBehaviour.t(),
           port: :inet.port_number(),
           use_ssl?: boolean(),
-          name: atom() | nil
+          name: atom() | nil,
+          lambda: function() | nil
```

Comment:
how about some more precise type? 
```suggestion
          lambda: ((server :: pid(), app :: String.t(), stream_key :: String.t() ) -> any())| nil
```